### PR TITLE
Fix ci and test

### PR DESCRIPTION
**Pull request by kyp76**
Tests are broken due a new version of httpretty.
Add pytest fixture to avoid error due to missing proxy set.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import pytest
+import os
+
+
+@pytest.fixture(scope="session", autouse=True)
+def set_proxy():
```
Comment (bew) [inline tests/conftest.py:6]: Why is this necessary?

Comment (kyp76) [reply tests/conftest.py:6]: To avoid error on test. if environment proxy is set by mistake, test will failed.
the scope of this feature is the session of pytest and it is used by default (autouse=True)
